### PR TITLE
fix executor selection in Manx terminal

### PR DESCRIPTION
**Pull request by khyberspache**
- Procedure selection will populate the correct executor/platform combination

### CODE REVIEW

File: static/js/terminal.js [hunk 1]
```diff
@@ -180,9 +180,14 @@ function filterProcedures() {
...
     function displayProcedure(data){
-        let a = data[0];
-        term.write(atob(a.test));
-        input = atob(a.test);
+        for (let ab of data) {
+            let agent = $('#session-id option:selected');
+            if (ab.platform === agent.data("platform") && agent.data("executors").includes(ab.executor)) {
+                term.write(atob(ab.test));
+                input = atob(ab.test);
+                break;
```
Comment (privateducky) [inline static/js/terminal.js:188]: it seems that we should print something if there is no match

Comment (privateducky) [reply static/js/terminal.js:188]: or is the data filtered before getting here, so there's really no chance of that?

Comment (khyberspache) [reply static/js/terminal.js:188]: I haven't dug deep enough to see if the data is getting filtered. I can poke

Comment (khyberspache) [reply static/js/terminal.js:188]: @privateducky Looks like it is filtered initially:
```javascript
restRequest('POST', {"paw": $('#session-id option:selected').data('paw')}, getAbilitiesCallback, '/plugin/manx/ability');
```

**Pull request merged**